2-virtual-module-source.md

```diff
@@ -136,9 +140,9 @@ direct === indirect; // true
 
 ## Dependencies
 
-This proposal depends on [Module and ModuleSource](./0.md) from the [Compartments
+This proposal depends on [Module and ModuleSource][0] from the [Compartments
 proposal](README.md) to introduce `ModuleSource`, and [ModuleSource
-analysis](./1.md) from the [Compartments proposal](README.md).
+analysis][0] from the [Compartments proposal](README.md).
```

Comment:
There are a bunch of places where you used `[0]` instead of the link to part 1 (grep for `analysis][0]`).